spec/spec_helper.rb

```diff
@@ -0,0 +1,25 @@
+require 'rspec'
+require 'selenium-webdriver'
+require 'capybara/rspec'
+require 'rack/jekyll'
+require 'rack/test'
+require 'pry'
+
+RSpec.configure do |config|
+  config.expect_with :rspec do |expectations|
+    expectations.syntax = :expect
+  end
+  # Configure JS Driver
+  Capybara.javascript_driver = :selenium_chrome_headless
+  # Configure Capybara to use Selenium.
+  Capybara.register_driver :selenium do |app|
+    # Configure selenium to use Chrome.
+    Capybara::Selenium::Driver.new(app, :browser => :chrome)
+  end
+
+  # Configure Capybara to load the website through rack-jekyll.
+  # (force_build: true) builds the site before the tests are run,
+  # so our tests are always running against the latest version
+  # of our jekyll site.
+  Capybara.app = Rack::Jekyll.new(force_build: true)
```

Comment:
`Capybara::Selenium::Driver.new(app, browser: :chrome)` - The community standards favour the JSON style.

Reply:
Is there a particular style-guide I can take a look at?

Reply:
https://github.com/rubocop-hq/ruby-style-guide

Or `gem install rubocop` and then run rubocop -a on any file you have changed. Am sure you can automatically put it in your workflow with vscode https://github.com/misogi/vscode-ruby-rubocop might be worth a look.